src/vmm/src/device_manager/persist.rs

```diff
@@ -84,6 +100,8 @@ pub struct ConnectedVsockState {
 #[derive(Versionize)]
 /// Holds the device states.
 pub struct DeviceStates {
+    /// Balloon device state.
+    pub balloon_device: Option<ConnectedBalloonState>,
```

Comment:
This needs bumping the snapshot version.

Any change to any `Versionized` struct requires bumping snapshot version.